src/main/java/com/moz/qless/Job.java

```diff
@@ -94,7 +93,7 @@
   protected String worker;
 
   @JsonCreator
-  Job(@JacksonInject("client") final Client client) {
```

Comment:
It looks like `@JacksonInject` no longer respects an argument for the name of the injected value. It looks like it only uses the type by default.

Reply:
I might have to look into how Jackson annotations work some time... did some quick google searches and didn't find any simple explanations...